### PR TITLE
Added 3 projects and 3 blogs for final PR

### PR DESCRIPTION
**Pull request by JowelTisso**
Aplogize for the late submit,
Kindly accept my final PR.
Thank you.

### CODE REVIEW

File: content/finalist/ttjc@finalist@JowelTisso/index.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+---
+title: Jowel Tisso
+type: "finalist"
+name: "Jowel Tisso"
+portfolio: "joweltissoportfolio.netlify.app"
+github-link: "https://github.com/JowelTisso"
+linkedin-link: "https://www.linkedin.com/in/joweltisso/"
+looking-for: "job"
+image-link: "https://avatars1.githubusercontent.com/u/52632590?s=400&u=591f8558da75762e94b5067612bf3543776e024f&v=4"
```
Comment (gkanishk) [inline content/finalist/ttjc@finalist@JowelTisso/index.md:9]: Replace the hyphen with an underscore in variables example:

```
github_link
linkedin_link
looking_for
image_link
```

File: content/finalist/ttjc@finalist@JowelTisso/index.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+It would be helpful if someone can help me in android Studio and React.
+
+### how to contact me?
+
+- [Twitter](https://twitter.com/joweltisso)
+- [LinkedIn](https://www.linkedin.com/in/joweltisso)
+
+### Projects
+
+Projects that i have completed during the job challenge :
```
Comment (gkanishk) [inline content/finalist/ttjc@finalist@JowelTisso/index.md:39]: I capital

**Comment by gkanishk** [comment]
Awesome! Welcome on-board!!!

Please share your details here  #1033

**Comment by JowelTisso** [comment]
Thank you...
But i have a doubt ,
I also have changed the variables in participants index file from '-' to ' _'
I feel i shouldn't have done that. Will there be any issue?

**Pull request merged**